# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mergeant/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual gcc-3.4 fixes.

  - Rene Rebe <rene@exactcode.de>

--- mergeant-0.12.1/src/interface_cb.c.orig	2004-05-10 15:59:39.247566552 +0200
+++ mergeant-0.12.1/src/interface_cb.c	2004-05-10 16:00:02.963961112 +0200
@@ -2210,5 +2210,6 @@
 						   gda_connection_get_errors (cnc));
 		break;
 	default :
+		;
 	}
 }
